services/cd-service/pkg/repository/repository.go

```diff
@@ -221,10 +236,17 @@ func (r *repository) ApplyTransformersInternal(transformers ...Transformer) ([]s
 }
 
 func (r *repository) ApplyTransformers(ctx context.Context, transformers ...Transformer) error {
-	commitMsg, state, err := r.ApplyTransformersInternal(transformers...)
+	commitMsg, state, err := r.ApplyTransformersInternal(ctx, transformers...)
+	//
 	if err != nil {
 		return err
 	}
+	if ddMetrics != nil {
```

Comment:
this line does not need to be here, `UpdateDatadogMetrics` takes care of the check